advanced/is-pattern-2/templates/analytics-worker/identity-server-analytics-worker-statefulset.yaml

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "fullname" . }}-analytics-worker-statefulset
+  namespace : {{ .Release.Namespace }}
+spec:
+  replicas: {{ default 1 .Values.wso2.deployment.wso2isAnalyticsWorker.replicas }}
```

Comment:
@darsham3 no need to define `default` values at the source level. We can depend on the values provided at the `values.yaml` file, as the default values.

Please apply the same to other values, where necessary.